base/stat.jl

```diff
@@ -53,8 +53,8 @@ StatStruct(buf::Union{Vector{UInt8},Ptr{UInt8}}) = StatStruct(
     ccall(:jl_stat_size,    UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blksize, UInt64,  (Ptr{UInt8},), buf),
     ccall(:jl_stat_blocks,  UInt64,  (Ptr{UInt8},), buf),
-    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf),
-    ccall(:jl_stat_ctime,   Float64, (Ptr{UInt8},), buf),
+    ccall(:jl_stat_mtime,   Float64, (Ptr{UInt8},), buf) |> Dates.unix2datetime,
```

Comment:
The `StatStruct` type needs to be changed to reflect this. Though I wonder which solution is best:
- change the struct fields to be `DateTime`
- keep the struct as is, and only change the `mtime`/`ctime` accessors to return a `DateTime` object

@vtjnash What did you have in mind when you filed that issue?